sdk-java/src/main/java/ly/count/sdk/java/internal/Storage.java

```diff
@@ -15,7 +13,11 @@
 
 public class Storage {
 
-    private static final Tasks tasks = new Tasks("storage", null);
+    private static Tasks tasks;
```

Comment:
When sdk stop called, this task instance is not cleaning. It stops its executor service. When you re-init Countly during app running, storage tasks give error like executor service is not running. To prevent that, this task service should be created in init